lib/DataRepeater.js

```diff
@@ -471,6 +473,17 @@ var DataRepeater = module.exports = kind(
 				p.removeListener(e, this._handlers[e]);
 			}
 		}
+		if(c && this.selectionProperty) {
+			var selectionProperty = this.selectionProperty,
```

Comment:
Good idea.  I have done so, and moved (and renamed) the local shadow of selectionProperty